### ISSUE TITLE
Implement GUI

### ISSUE THREAD

**Issue by shahar0809** [CS-FinalProject/StyleTransfer#1]
Implement GUI for the project. Basic functionality:

# Choose Image
## Drop window
A little box in which the user can drop boxes into

![like this](https://i0.wp.com/css-tricks.com/wp-content/uploads/2015/11/drag-drop-upload-1.gif?ssl=1)

Not necessarily the same style

## Upload Image
Just a button that allows to browse the files for an image (supported extensions: png, jpg, jpeg)

## Choose from sample images
Allows user to choose image from a gallery of images

# Calling model
basically just calling the model from a button, but need to design it.

design is in drawio file
